examples/bpk-component-graphic-promotion/stories.js

```diff
@@ -30,11 +31,12 @@ import {
 } from './examples';
 
 storiesOf('bpk-component-graphic-promotion', module)
-  .add('Default styling', NoStyleExample)
-  .add('Default', DefaultExample)
+  .add('Default no background image', NoStyleExample)
+  .add('Default with background image', DefaultExample)
   .add('Center Aligned', CenterAlignedExample)
   .add('Right Aligned', RightAlignedExample)
   .add('Inverted Portrait Mode', InvertedPortraitExample)
   .add('Minimalistic', MinimalisticExample)
   .add('Non-Sponsored', NonSponsoredExample)
-  .add('Visual test', MixedExample);
+  .add('Mixed', MixedExample)
```

Comment:
Given we have changed up the visual test and the other stories exist individually already we won't need to have this story anymore so can be removed 😄